src/databases/schema/users.js

```diff
@@ -21,10 +22,23 @@ const UserSchema = new mongoose.Schema({
     type: String,
     required: true,
   },
-  location: [String],
-  friends: [],
-  pending: [],
-  requests: [],
+  stat: {
+    userStatus: {
+      type: String,
+      default: "Offline"
+    },
+    lastUpdated: {
+      type: Date,
+      default: () => Date.now(),
+    }
+  },
+  location: {
+    longitude: String,
+    latitude: String
+  },
+  friends: [mongoose.SchemaTypes.ObjectId],
+  pending: [String],
+  requests: [String],
```

Comment:
Are these supposed to be `ObjectId`'s?

Reply:
No, they are strings that contain the usernames of possible friends